packages/generate-persisted-query-manifest/package.json

```diff
@@ -40,6 +40,7 @@
     "graphql": "14.x || 15.x || 16.x"
   },
   "devDependencies": {
+    "@apollo/client": "3.9.5",
```

Comment:
should we update the peer dep while we're at it?  maybe we don't really need to leave in support for a random alpha now that 3.8.0 is old news

Reply:
Ya good call! Wasn't sure how much I should clutter this PR up, but I think thats a good change.